fdbserver/RestoreRoleCommon.actor.cpp

```diff
@@ -56,11 +56,17 @@ void handleFinishRestoreRequest(const RestoreVersionBatchRequest& req, Reference
 }
 
 void handleInitVersionBatchRequest(const RestoreVersionBatchRequest& req, Reference<RestoreRoleData> self) {
-	self->resetPerVersionBatch();
-	TraceEvent("FastRestore")
-	    .detail("InitVersionBatch", req.batchID)
-	    .detail("Role", getRoleStr(self->role))
-	    .detail("Node", self->id());
+	// batchId is continuous. (req.batchID-1) is the id of the just finished batch.
+	self->versionBatchId.whenAtLeast(req.batchID - 1);
```

Comment:
this returns a future, we need to wait on it

Reply:
Maybe whenAtLeast as a good candidate for [[nodiscard]] then?